tests/test_recomb_collector.py

```diff
@@ -30,32 +32,36 @@ def check_tables(args):
         assert(ch < args.num_nodes)
 
 
-def test_simupop_runs():
-    for popsize in [5, 10, 20]:
-        print("Popsize: ", popsize)
-        generations = 3
-        # replications = 1
-        nsamples = 2
-        length = 10
-        nloci = 5
-        locus_position = list(range(0, length, int(length/nloci)))
-        recomb_rate = 0.05
-
-        rc = RecombCollector(
-                nsamples=nsamples, generations=generations, N=popsize,
-                ancestor_age=10, length=length, locus_position=locus_position)
-
-        init_geno = [sim.InitGenotype(freq=[0.9, 0.1], loci=sim.ALL_AVAIL)]
-
-        id_tagger = sim.IdTagger(begin=0)
-        id_tagger.reset(startID=1)  # must reset - creating a new one doesn't
-
+@pytest.fixture(scope="function", params=[
+    lambda recombinator, popsize: sim.RandomMating(
+        ops=[
+            sim.IdTagger(),
+            recombinator
+        ]),
+    # Overlapping generations mating system
+    lambda recombinator, popsize: sim.HeteroMating([sim.RandomMating(
+        ops=[
+            sim.IdTagger(),
+            recombinator
+        ]),
+        sim.CloneMating()],
+        subPopSize=popsize * 2)
+    ])
```

Comment:
Given the stage the rest of the code is at, I think this is as detailed of an overlapping generation test we want here in `ftprime`. It passes. 

At some point (in a future PR) it'd be good to separate tests of various aspects of `recomb_collector` into clearer units.